plugin/src/main/groovy/com/novoda/gradle/command/DemoModeTask.groovy

```diff
@@ -0,0 +1,64 @@
+package com.novoda.gradle.command
+
+import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.tasks.TaskAction
+
+class DemoModeTask extends AdbTask {
+
+    boolean enable
```

Comment:
Seems this is not really a property. Could it be a constructor parameter instead?

Reply:
Couldn't find an easier way to use constructor to create tasks in gradle. But in any case, will remove this and create 2 separate tasks.

Reply:
Tasks are separated. https://github.com/novoda/gradle-android-command-plugin/pull/125/commits/d2690644917ef57bfb914435c3633e923927f3d3